src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -1201,8 +1205,10 @@ void core::start_oxenmq() {
                 false,
                 m_pulse_thread_id);
         m_omq->add_timer([this]() { this->check_service_node_time(); }, 5s, false);
+        m_omq->add_timer([this]() { this->check_service_node_ip_address(); }, 15min, false);
     }
     m_omq->start();
+    check_service_node_ip_address();
```

Comment:
This forces an IP check after initialization instead of deferring it 15 minutes.

Reply:
> This forces an IP check after initialization instead of deferring it 15 minutes.

This is a useful comment that you should add to the code instead of having it get lost in a github comment

Reply:
agreed and added